.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,133 @@
+name: Test coverage
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  e2e-coverage:
+    name: e2e-test-coverage
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
just realised this is actually not needed really, should remove from `pr.yml` as well, can we do it as part of this PR?

Reply:
I can remove it here, would keep the other workflow change dedicated not to mix too many things.